doctor/routetable/routetable_bsd.go

```diff
@@ -0,0 +1,256 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build darwin || freebsd
```

Comment:
probably openbsd as well (we have a small but mighty contingent of OpenBSD users)

Reply:
I don't have an OpenBSD system to test on, but I will set up a VM and test this package as a follow-up PR 👍